dist/profile/manifests/usage.pp

```diff
@@ -164,7 +165,8 @@
     override        => ['All'],
     docroot         => $docroot,
     error_log_file  => "${usage_fqdn}/error_nonssl.log",
-    access_log_pipe => "|/usr/bin/rotatelogs ${apache_log_dir}/access_${::ipaddress}_nonssl.log.%Y%m%d%H%M%S 86400",
+    access_log_pipe => "|/usr/bin/rotatelogs ${apache_log_dir}/access_${facts['facts['networking']['ip']']}_nonssl.log.%Y%m%d%H%M%S 86400",
```

Comment:
Why this one is different?

Reply:
oh, sounds like it is the puppet autoformatting in my browser. Lemme revert

Reply:
fixed. thanks for the review!